azure-functions/acbs-function/helpers/currency.js

```diff
@@ -1,5 +1,27 @@
 const stripCommas = (value) => String(value).replace(/,/g, '');
-const to2Decimals = (value) => Number(Number(stripCommas(value)).toFixed(2));
+/**
+ * @description Rounds a number to 2 decimal places.
+ * Number cooersion is as per the built in Number function,
+ * with the exception of removing commas from strings before parsing.
+ * @param {unknown} value value to round to 2 decimal places
+ * @returns {number} value rounded to 2 decimal places
```

Comment:
`toFixed` returns a string 😛

Reply:
The call of `toFixed` is wrapped in the `Number` constructor, so this does return a `number`

Reply:
I see thanks @JoshBinns2000 sorry didn't realise